src/components/login/Login.tsx

```diff
@@ -1,70 +1,69 @@
 import React from 'react';
 import { inject, observer } from 'mobx-react';
+import { Redirect } from 'react-router-dom';
+import { Location } from 'history';
+import hslLogo from '~/assets/hsl-logo.png';
+import loginBackground from '~/assets/loginBackground.jpg';
```

Comment:
The login picture is 2500 px wide, we could probably make it a bit more optimized by having a slightly smaller image. The image is so blurry, so I don't think anyone notices if it would have one fourth of the current resolution.

Reply:
Resized now to 616 x 322, size dropped about 50%